assets/apps_linux.json

```diff
@@ -13,7 +13,8 @@
 		"name":"Steam BigPicture",
 
 		"output":"steam.txt",
-		"detached":["setsid steam steam://open/bigpicture"]
+		"detached":["setsid steam steam://open/bigpicture"],
+		"image-path":"./assets/steam.png"
```

Comment:
Are we sure we can actually use the steam image? Steam is a trademarked logo after all

Reply:
Ironically, the Steam artwork used by GFE does not follow their guidelines, haha.

Perhaps, the colors need to be adjusted a bit. Seems they want specific colors. Although I doubt they would throw a fit over this. Below is the colors they use in the guidelines.

Also we need to add this attribution somewhere... maybe in the webui?

```txt
©2018 Valve Corporation. Steam and the Steam logo are trademarks and/or
registered trademarks of Valve Corporation in the U.S. and/or other countries. All
rights reserved.
```
![image](https://user-images.githubusercontent.com/42013603/152360019-4d99b1fe-e446-43a8-92d5-1c8c201afc1a.png)

Reply:
We can simply add this onto a NOTICE file on the root of the repo and then bundle it inside the zip file on the release, we should be okay then

Reply:
ok

Reply:
Done. CMake also copy the file into build directory, but don't know if it's useful.
I've some questions :
- Linux packages : Where do we put the file ?
- Is there other package to modify ?

Reply:
Packaging is done by the GitHub actions, I don't remember the exact details but it definitely can be edited to include the file in the artifacts (I don't know if it should be done here or in another PR though)

Reply:
Let's take care of it in another PR, probably would be good to add the LICENSE too.